### DISCUSSION TITLE
How to apply settings to all folders and files except root folder?

### DISCUSSION THREAD

**Discussion by windblower420** [SebastianMC/obsidian-custom-sort#48] [Q&A] [answered]
hello I finally achieved what I wanted ! im using 

```
---
sorting-spec: |
 target-folder: //*
 /:files
  > a-z
 /folders
  > a-z
---
```

but I don't want my root files to be affected by the sorting. how do I have the root files ignored but not the children folders and the content in them

**Comment by SebastianMC** [comment, marked as answer]
Hi!

the short answer is: **specify the sorting rule for the root folder explicitly**

```
---
sorting-spec: |
 target-folder: //*
 /:files
  > a-z
 /folders
  > a-z

 target-folder: /
 sorting: standard
---
```

Explanation: **specifying the target folder by name has always a higher priority than any wildard-pattern-based sorting specification.**

In your specific example, the line ``` target-folder: //*``` sets the sorting rules for all folders (including root) of the vault. Then, adding a line ``` target-folder: /``` sets the sorting rules for the root ```/``` folder specifically, overriding the pattern-based configuration.

I've used the ``` sorting: standard``` in the example - in result, the files and notes in the root folder will be managed by the standard obsidian sorting mechanism (the `Change sort order` toolbar button in File Explorer). You can obviously specify any other sorting rule supported by the plugin instead.

BTW you can use the simpler syntax ``` target-folder: /*``` instead of ``` target-folder: //*```. They are equivalent.
